src/styles/utils/_variables.scss

```diff
@@ -0,0 +1,11 @@
+$cell-size: 100px;
+$move-cell-size: 20px;
+$days-of-week: (
+  "mon" 0,
+  "tue" 103px,
+  "wed" 206px,
+  "thu" 309px,
+  "fri" 412px,
+  "sat" 515px,
+  "sun" 618px
+);
```

Comment:
Strange solution, better set here modifier for day number and save one cell size, than multiply them, I don't think that it's a good idea to hardcode size for every case 🙂